hw/ip_templates/ac_range_check/dv/env/ac_range_check_env_pkg.sv.tpl

```diff
@@ -34,6 +34,8 @@ package ac_range_check_env_pkg;
   parameter uint   NUM_ALERTS                 = 2;
   parameter string LIST_OF_ALERTS[NUM_ALERTS] = {"recov_ctrl_update_err", "fatal_fault"};
   parameter uint   NUM_RANGES                 = ${num_ranges};
+  parameter uint   NUM_ROLES                  = ${num_roles};
+  parameter uint   ROLEWIDTH                  = ${nr_role_bits};
```

Comment:
Can we use `2**top_racl_pkg::NrRaclBits` and `top_racl_pkg::racl_role_t` for that? That way we can remove that racl param to the ipgen IP.